docs/API.md

```diff
@@ -32,7 +32,7 @@ source = <<-EOS
 EOS
 
 reporter = Reek::Report::TextReport.new
-examiner = Reek::Examiner.new(source)
+examiner = Reek::Examiner.new source
```

Comment:
This is a super minor nitpick, but while I used to preach the Seattle-style ‘all parentheses must go!’ approach in the past I now got used to – and like _very much_ – the idea of keeping the parentheses when returning a value. I also think that keeping the parentheses here would make this a tiny bit clearer/unambiguous, which would be most welcome in API docs. :)